### DISCUSSION TITLE
Configuring EV1527 Decoder

### DISCUSSION THREAD

**Discussion by burnski666** [merbanan/rtl_433#3037] [General]
I'm using https://github.com/merbanan/rtl_433/blob/master/conf/EV1527-4Button-Universal-Remote.conf to pull in data from a couple of 433MHz remotes I use around the house. Im trying to break them out though so they arent all publishing to the same MQTT topic. In the notes for the decoder it says "The shown example can distinguish between multiple remotes (two in this case). If this is not desired, a simple "match=abcde" can limit the detection of one unique EV1527 code." however it seems wherever i put this into the decoder just causes my SDR to stop working. 

Here are my 2 remotes that im trying to publish to separately: 

![b69c05d94bc2e1437e208545b514a73372dac9f0_2_1035x625](https://github.com/user-attachments/assets/78c674fe-7636-42cb-b85a-057102368e94)
![c3591ae0fc36fe9fc5b2cef21a1f22099624ef8f](https://github.com/user-attachments/assets/38351ce7-04b0-4771-90fa-538a1d55bca5)

Can anyone point me to how/where I would add said line? thanks.

**Comment by zuckschwerdt** [comment]
Start by modifying the first get line in EV1527-4Button-Universal-Remote.conf. Use `code` output like it's shown from that getter as parameter (e.g. 619871)  to distinguish remotes. Note that it's decimal for the first 20 bits of the `data` output (e.g. 9755f here).
If that works well you can clone the flex decoder, change the name in each one and add e.g. a `match=9755f,` somewhere.

**Reply by burnski666** [reply]
Dude you are my hero, this has been my problem was that I wasn't putting the code in HEX. Just tried it and it's flawless. Thank you so much
